matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/common/CommonTestHelper.kt

```diff
@@ -50,6 +50,16 @@ import java.util.UUID
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 
+internal fun withTestHelpers(context: Context, block: (CryptoTestHelper) -> Unit) {
```

Comment:
if we follow the conventions from the coroutine-test api then this could be `runCryptoTest`